### ISSUE TITLE
.NET Core Sdk app/lib require a build before intellisense works

### ISSUE THREAD

**Issue by enricosada** [ionide/ionide-vscode-fsharp#312]
Opening a new project, or cloned, or just restored, the intellisense doesnt work.

Affect projects created with the .NET Core Sdks:

- `project.json` based (`preview2`, `preview2.1`)
- msbuild based, so fsproj (`preview3`, `preview4`, ..)

### Repro

```
dotnet new -l fsharp
dotnet restore
code .
````

### Expected

Intellisense works

### Actual

Intellisense doesnt work.

### Workaround

Do a `dotnet build` **BEFORE** opening VS Code.

So the following works:

```
dotnet new -l fsharp
dotnet restore
dotnet build
code .
```

### Why/TODO

That happen becauseFSAC read the response file generated after build to initialize FCS.
So no build, no response files.

Ideas for minor improvements /cc @Krzysztof-Cieslak @cloudRoutine for ideas

- [ ] watch after build if the response file exists, and reload it ?
- [ ] reinitialize FCS after build ?
- [ ] watch the response file found, and reinitialize if changed?

**Comment by czifro** [comment]
I figured I would comment here rather than [xunit/xunit#859](https://github.com/xunit/xunit/issues/859) since it seems more appropriate, but I upgraded dotnet core to preview4-004233 and did `dotnet restore` and `dotnet build` before opening vscode, but it still is showing errors where there is none, mainly references to packages or modules within the root namespace. Is there a step I am missing?

Additional Info:
```
VS Code Version => 1.8.1
Ionide-fsharp Version => 2.14.0
dotnet version => preview4-004233
OS => macOS Sierra 10.12.1
```

**Comment by enricosada** [comment]
@czifro can you check your fsproj is similar to these generated by `dotnet new -l fsharp` ? Or can i see the proj?

**Comment by czifro** [comment]
Here is my fsproj: https://github.com/czifro-tech/mudt/blob/master/fsharp/src/MUDT.fsproj

**Comment by enricosada** [comment]
Problem is you are using an fsproj for preview3. 
Please change it like preview `dotnet new -l fsharp -t lib` ( so Sdk attribute, no imports and use fsharp.net.sdk 1.0.0-beta-*)
I'll try to fix it tomorrow If you cannot

**Comment by czifro** [comment]
That fixed it. Thank you!

**Comment by enricosada** [comment]
Done! fixed with 2.25.1

just `dotnet restore` is required.
